### ISSUE TITLE
Unbinded LDAP connection

### ISSUE THREAD

**Issue by thorsteneckel** [zammad/zammad#1712]
### Infos:

* Used Zammad version: 2.2
* Installation method (source, package, ..): git
* Operating system: any
* Database + version: any
* Elasticsearch version: any
* Browser + version: any

### Expected behavior:

* It should be possible to connect to LDAP servers that don't have/need a binded connection.

### Actual behavior:

Zammad requires a binded connection. When trying to connect to a OpenDirectory on macOS(10.12.6) and Server.app(5.3.1) the connection fails:

```
Can't bind to '*SERVER*', 34, Invalid DN Syntax (Exceptions::UnprocessableEntity)
/opt/zammad/lib/ldap.rb:138:in `binded_connection'
/opt/zammad/lib/ldap.rb:122:in `connect'
/opt/zammad/lib/ldap.rb:38:in `initialize'
```

### Steps to reproduce the behavior:

* Try to connect to e.g. a OpenDirectory on macOS(10.12.6) and Server.app(5.3.1) .

**Comment by KentaroGraphics** [comment]
It was resolved after updating to Zammad 2.2!! Thanks a lot.

**Comment by thorsteneckel** [comment]
Glad to hear! Thanks for your feedback.
